src/main/java/org/broadinstitute/consent/http/db/DataAccessRequestDAO.java

```diff
@@ -36,6 +36,26 @@
           + "  AND draft != true ")
   List<DataAccessRequest> findAllDataAccessRequests();
 
+
+  /**
+   * Find all non-draft DataAccessRequests for the given datasetId including only the dar data needed for the metrics page
+   *
+   * @return List<DataAccessRequest>
+   */
+  @RegisterRowMapper(DataAccessRequestMapper.class)
+  @SqlQuery(
+      "SELECT id, reference_id, draft, user_id, create_date, sort_date, submission_date, update_date, "
+      + " ((data #>> '{}')::jsonb->>'projectTitle') AS project_title, "
+      + " ((data #>> '{}')::jsonb->>'investigator') AS investigator, "
+      + " ((data #>> '{}')::jsonb->>'nonTechRus') AS nonTechRus, "
+      + " ((data #>> '{}')::jsonb->>'non_tech_rus') AS non_tech_rus, "
+      + " ((data #>> '{}')::jsonb->>'darCode') AS darCode, "
+      + " ((data #>> '{}')::jsonb->>'dar_code') AS dar_code "
```

Comment:
I'm concerned about this approach. In our current reconstruction of the jsonb data, we run it through regexes to ensure it is convertible to a `DataAccessRequestData` object. All of the logic is centralized in that class. I think we should stick with that approach instead of trying to pull these values out individually. Selecting for the camel cased and snake cased names will becomes unmanageable over time.

Reply:
Okay, so then, do we want to retrieve the whole dar object? I was doing this in response to Justin's feedback, so the response wouldn't be large and bulky. If we just want to get the entire dar, then before returning it do we just construct a new object with only the info we need/remove the information we don't need?

Reply:
The response is a `DatasetMetrics` object that doesn't necessarily need to have a list of full DARs, it could be a different object altogether with just the info that you want. The issue with this code as it stands now is that it sets up two different ways of parsing the json which introduces room for bugs and inconsistencies. If instead you used the current mapping for DAR info and then grab only the fields you want for a specific metric, it would be more efficient in the response.

Reply:
Okay that makes sense thanks!